src/components/button-minor/button-minor-test.stories.tsx

```diff
@@ -10,7 +10,7 @@ import {
 
 export default {
   title: "Button Minor/Test",
-  includeStories: ["DefaultStory"],
+  excludeStories: ["ButtonMinorCustom", "ButtonMinorDifferentTypes"],
```

Comment:
**question**: what was the reason for excluding these stories? The `ButtonMinorDifferentTypes` story especially seems useful to keep

Reply:
Good point 👍🏼 These stories appear to be old artifacts for our Cypress/Playwright tests.

Before we migrated all the Cypress tests to TypeScript, we decided to temporarily place the mock components within the `test-stories.tsx` files so we could type-check them. We also have these mock components defined in the `components.test-pw.tsx` file.

If having these mock components as Storybook test stories would be valuable, I'm happy to include them again 😄